package.json

```diff
@@ -64,8 +64,9 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "build-sw": "node ./src/custom-sw/sw-env.js && node ./src/custom-sw/sw-build.js",
+    "translation-sync": "node ./src/lib/i18next/getResources.js",
     "clean-cra-sw": "rm -f build/precache-manifest.*.js && rm -f build/service-worker.js",
-    "build": "react-scripts build && npm run build-sw && npm run clean-cra-sw",
+    "build": "react-scripts build && npm run build-sw && npm run translation-sync && npm run clean-cra-sw",
```

Comment:
Shouldn't the `npm run translation-sync` run before the `react-scripts build`? Or else the `build/` dir produced for production would use the `resources.js` from before the sync right?

You can test this by building once after pointing to the `Mee Panyar Translations` base ID (I changed the table name and column names to work with the script). Then serving the production build via `serve -s build`. If you go to the site and do `devtools --> source` and search for `resources.js` you should see that the translations haven't updated to the translations from the new base ID

The production builds with the stale file (before the sync occurs), and so `build` needs to run twice for the production build to get the new translations.